### ISSUE TITLE
Umowy własne - przewoźnik, jak nadać zlecenie transportowe za pomocą API

### ISSUE THREAD

**Issue by OmegaVZ** [allegro/allegro-api#9337]
Witam,
Jak złożyć zapytanie o nadanie listu za pomocą kuriera zaprogramowanego na umowie własnej, a dokładniej jak pozyskać jego ID.
Do złożenia zlecenia z tego co rozumiem potrzebuje **credentialsId**.
Jak to jest opisane na https://developer.allegro.pl/tutorials/jak-zarzadzac-przesylkami-przez-wysylam-z-allegro-LRVjK7K21sY
"credentialsId":"c9e6f40a-3d25-48fc-838c-055ceb1c5bc0", - identyfikator umowy własnej; wymagany, jeżeli nadaje
Próbowałem zapytania, aby pozyskać credentialsId
https://api.allegro.pl/shipment-management/delivery-services
Nie zwraca takich informacji, mimo, że poprawnie skonfigurowałem i przetestowałem przewoźnika DPD na umowie własnej.
{
  "services": [
    {
      "id": {
        "deliveryMethodId": "c3066682-97a3-42fe-9eb5-3beeccab840c", - id usługi dostawy
        "credentialsId": "c9e6f40a-3d25-48fc-838c-055ceb1c5bc0" - id umowy własnej (jeśli brak, zwracamy "null")
      },
Według opisu [GET /shipment-management/delivery-services](https://developer.allegro.pl/documentation/#operation/getDeliveryServices), odpowiedź powinna zwrócić informacje, ale nie otrzymuje ich, poza konfiguracjami nadań Allegro.

Zamówienie zwraca tylko
        "method": {
            "id": "259b5c7a-9056-4c74-80ec-9bb2978cf293",
            "name": "Kurier DPD pobranie"

Proszę o informacje jak pozyskać credentialsId.

**Comment by PrzemyslawLukanowski** [comment]
Czy udało Ci się poprawnie dodać umowę własną przez stronę https://allegro.pl/moje-allegro/sprzedaz/zamowienia/ustawienia?

**Comment by OmegaVZ** [comment]
Tak, mam dodaną umowę własną DPD i przetestowałem nadanie przesyłek za jej pomocą, tylko nie wiem jak podejść do automatyzacji za pomocą API bo nie mogę znaleźć lub nie otrzymuje informacji wymaganej do takiego nadania wspomnianej w dokumentacji credentialsId.

![obraz](https://github.com/allegro/allegro-api/assets/122499590/447a077a-efda-47cd-a219-c9f215e15a61)

**Comment by PrzemyslawLukanowski** [comment]
Podaj proszę trace-id dla wywołania GET /shipment-management/delivery-services, gdzie w odpowiedzi nie zwróciliśmy tej umowy.

**Comment by OmegaVZ** [comment]
trace-id: 4a5b2929e9b7ff5f

**Comment by PrzemyslawLukanowski** [comment]
Twój token nie posiada kontekstu użytkownika. Musiałeś wygenerować go w ramach autoryzacji [client_credentials](https://developer.allegro.pl/tutorials/uwierzytelnianie-i-autoryzacja-zlq9e75GdIR#clientcredentials-flow) zamiast [code](https://developer.allegro.pl/tutorials/uwierzytelnianie-i-autoryzacja-zlq9e75GdIR#authorization-code-flow) lub [device](https://developer.allegro.pl/tutorials/uwierzytelnianie-i-autoryzacja-zlq9e75GdIR#device-flow). W naszej dokumentacji znajdziesz odpowiednie oznaczenie, jakiego rodzaju autoryzacji wymagamy:
<img width="455" alt="Zrzut ekranu 2024-07-4 o 13 01 41" src="https://github.com/allegro/allegro-api/assets/51473661/5ff16c99-ba57-49a7-96b5-aecf6b14d914">
<img width="526" alt="Zrzut ekranu 2024-07-4 o 13 01 51" src="https://github.com/allegro/allegro-api/assets/51473661/fccd6ff3-ff2e-41f1-b639-5b0ef6540279">
